src/Views/RangeSlider.elm

```diff
@@ -97,6 +98,54 @@ yarnSize config =
         }
 
 
+type alias PhysicalDurabilityConfig msg =
+    { id : String
+    , update : Maybe Unit.Durability -> msg
+    , value : Unit.Durability
+    , toString : Unit.Durability -> String
+    , disabled : Bool
+    }
+
+
+physicalDurability : PhysicalDurabilityConfig msg -> Html msg
+physicalDurability config =
+    let
+        customLayout : { id : String, label : String, attributes : List (Attribute msg) } -> Html msg
+        customLayout { id, label, attributes } =
+            div [ class "RangeSlider row", style "flex-grow" "1" ]
+                [ div [ class "col-xxl-2" ]
+                    [ Html.label [ for id, class "form-label text-nowrap mb-0" ]
+                        [ text label ]
+                    ]
```

Comment:
Why did we need a custom layout here exactly?

Reply:
Because the other one is used for this case

![Screenshot 2024-09-09 at 11-18-11 Simulateur Ecobalyse](https://github.com/user-attachments/assets/a6946270-2162-4c6d-9719-a158428fa75d)

Cols are not the same, label size is smaller too and without the custom `flex-grow` the slider doesn't shrink as it should, but if you have another solution I'm ok for it :)

Reply:
I'm gonna check the branch and propose a patch about that

Reply:
Ok I finally took the libverty to push a slight refactor in https://github.com/MTES-MCT/ecobalyse/pull/721/commits/50976cf1035f4f0f87f4492d7d36aebb0fca4746